src/unix/stream.c

```diff
@@ -795,7 +799,7 @@ static void uv__write(uv_stream_t* stream) {
     msg.msg_iovlen = iovcnt;
     msg.msg_flags = 0;
 
-    msg.msg_control = (void*) scratch;
+    msg.msg_control = (void*) &scratch.alias;
```

Comment:
Wouldn't `msg.msg_control = &scratch[0];` work?  I remember the `(void*) scratch` was to work around a false positive in an old version of apple-gcc but that's probably obsolete now.